frontend/src/components/SystemsTable/SystemTableTools.tsx

```diff
@@ -205,13 +234,16 @@ export function SystemTableTools({
         {deleteButton}
       </Space>
       <Space style={{ width: "fit-content", float: "right" }}>
-        <Radio.Group
-          options={showMineOptions}
-          onChange={({ target: { value } }) => onChange({ showMine: value })}
-          value={value.showMine}
-          optionType="button"
-          buttonStyle="solid"
-        />
+        {mineVsAllSystemsToggle}
+        {/* <Tooltip title={remindLogInMessage}>
```

Comment:
We can remove the dead code.

Reply:
I think this is deleted in the succeeding [commit](https://github.com/neulab/explainaboard_web/pull/321/commits/3313dace9f40308c2ad83120afb9f7145f2002b9)